Makefile

```diff
@@ -22,6 +22,11 @@ deps:
 	npm install
 	node_modules/gulp/bin/gulp.js
 
+deps3:
+	pip3 install -r requirements.txt
```

Comment:
I didn't realize it was so easy to do virtual environments in python! I'll clean it up a bit later